File: GuessNumber.py
import random
guessNumber = random.randrange(1, 101)
# print(guessNumber) # Debugging

i = 0
while True:
    try:
        guess = int(input("Guess The Number: "))
        if guess > guessNumber:
            print("Too high! Try again.")
            i = i + 1
        elif guess < guessNumber:
            print("Too low! Try again.")
            i = i + 1
        elif guess == guessNumber:
            i = i + 1
            print("Congratulations! You guessed the number in", i, "tries!")
            break
    except ValueError:
        print("Number not recognized! Try again.")

File: guess_number.rb
secret_number = rand(1..100)

puts "I'm thinking of a number between 1 and 100"
puts "Can you guess what it is?"

loop do
  print "Enter your guess: "
  guess = gets.chomp.to_i

  if guess == secret_number
    puts "You got it! The number was #{secret_number}."
    break
  elsif guess > secret_number
    puts "Too high. Try again."
  else
    puts "Too low. Try again."
  end
end
